_pages/general-information-and-resources/tech-policies/bug-bounty.md

```diff
@@ -8,15 +8,86 @@ questions:
 TTS administers a [Bug Bounty Program](https://hackerone.com/tts) for the public
 to report vulnerabilities in TTS systems.
 
-## Adding a team member to the Bug Bounty program
+## How it works
+
+1. Security researchers submit reports based on program policy
+1. H1 staff triage the report
+1. You confirm the report and award the bounty
+1. You publish a fix to production and resolve the report
+1. (optional) The researcher confirms the fix
+1. (optional) Disclose the report
+
+Security researchers review the [program policy](https://hackerone.com/tts) for
+what systems are in scope of the Bug Bounty Program and rules of behavior.
+
+Researchers report to the Bug Bounty Program where H1 staff do the initial
+triage.
+
+Once Triaged, the report will be assigned to your group (H1 term for team) for
+you to confirm. You can ask H1 staff or the researcher for any
+clarification, or help if you're having trouble reproducing the issue.
+
+Once confirmed, you can [award the bounty](). You don't need to wait until the
```

Comment:
@adborden what link goes here?

Reply:
Oops 😅  should link to the heading below about awards. Fixed now. I was surprised that didn't get caught by the link checker. I guess it's technically a valid link.